### PR TITLE
[Added] Market making section

### PR DESCRIPTION
**Pull request by 0amirk**
[Added] market section as mentioned in #5.

- [x] did not hard code data
- [x] added necessary space and padding
- [x] images were taken as svgs

desktop view.
Screenshot 2023-09-03 132536
tablet view.
Screenshot 2023-09-03 132553
mobile view.
Screenshot 2023-09-03 132606

### CODE REVIEW

File: src/components/Market section/index.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        src={gradient}
+        className="absolute md:-top-full  z-0  md:h-[2018.932px] -left-4 md:left-0 opacity-80"
+      />
+
+      {topArr.map((step, index) => (
+        <div
+          className="relative z-10 flex flex-col items-center mb-20 md:mb-32"
+          key={index}
+        >
+          <p className="mb-5 text-4xl font-medium md:text-5xl">{step.head}</p>
```
Comment (Bhargav-44) [inline src/components/Market section/index.jsx]: add text-center

File: src/components/Market section/index.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <div
+          className="relative z-10 flex flex-col items-center justify-around md:flex-row"
+          key={index}
+        >
+          <div className="text-left md:text-left">
+            <p className="max-w-sm mb-6 text-3xl leading-snug md:text-4xl">
+              {step.head}
+            </p>
+            <p className="max-w-sm mb-5 text-xl font-bold">{step.subHead}</p>
+            <p className="max-w-sm mb-6 text-xl">{step.description}</p>
```
Comment (Bhargav-44) [inline src/components/Market section/index.jsx]: add text-[#E5E5E5]

File: src/components/Market section/index.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import gradient from "/Market section/gradient.svg";
+import arrow from "/Market section/arrow.svg";
+import { topArr, bottomArr } from "./data";
+function Market() {
+  return (
+    <div className="bg-[#141619] text-white relative px-4 md:p-0 z-10">
```
Comment (Bhargav-44) [inline src/components/Market section/index.jsx]: change div to section

File: src/components/Market section/data.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import token from "/Market section/token.svg";
```
Comment (Rahilsiddique) [inline src/components/Market section/data.js]: use `camelCase` or something but dont put spaces in folder or file name
`Market Section` could be written as `MarketSection` or `marketSection`

**Comment by 0amirk** [comment]
Changes made.

File: src/components/marketSection/index.jsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
```
Comment (Bhargav-44) [inline src/components/marketSection/index.jsx]: no need to import React

**Pull request merged**